skipper/cli.py

```diff
@@ -258,7 +274,10 @@ def _prepare_build_container(registry, image, tag, git_revision=False):
 
     docker_file = utils.image_to_dockerfile(image)
     utils.logger.info("Building image using docker file: %(docker_file)s", dict(docker_file=docker_file))
-    runner.run(['docker', 'build', '-t', image, '-f', docker_file, '.'])
+    ret = runner.run(['docker', 'build', '-t', image, '-f', docker_file, '.'])
+
+    if ret != 0:
+        raise RuntimeError('Failed to build image: %(image)s', dict(image=image))
```

Comment:
how about removing the exception and writing

`exit('Failed to build image: %(image)s', dict(image=image))`

the program will exit immediately with the proper message, so you wouldn't have to manage the exceptions and logs...